refreshdata.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+""" Refresh data files for the Covid19 Mtl dashboard """
+
+import os
+import sys
+
+from datetime import datetime
+from argparse import ArgumentParser
+
+import requests # TODO: add these to requirements.in after merging with merwork
+import lxml
+import bs4
+
+# Data sources mapping
+# {filename: url}
+SOURCES = { #'cases.csv': '', 
+            #'cases_per1000': '', 
+            #'data_mtl': '',
+
+            # The main page for this dataset is https://www.inspq.qc.ca/covid-19/donnees 
+            'data_qc.csv': 'https://www.inspq.qc.ca/sites/default/files/covid/donnees/combine.csv'}
+DATA_DIR = '/tmp/data'
+NB_RETRIES = 3
+
+def fetch(url):
+    ''' Get the data at `url`.  Our data sources are notoriously unreliable, 
+    so we retry a few times. '''
+    for i in range(NB_RETRIES):
```

Comment:
I am surprised/disappointed that requests doesn’t provide anything for this.

Found a few posts about using urllib3 + request adapter for this: https://www.peterbe.com/plog/best-practice-with-retries-with-requests

That code snippet has been packaged by someone else: https://pypi.org/project/retry-requests/
(not sure about quality, stability, etc!)

Reply:
I think we can leave it like that for now.  It's a little ugly, but it's factored out and it does the right thing.